runtimes/dotnet-6.0/build.sh

```diff
@@ -28,16 +28,6 @@ build() {
         break
     done
 
-
-    case ${INTERNAL_RUNTIME_ENTRYPOINT##*.} in 
-        cs) write_cs_wrapper ;;
-        fs) write_fs_wrapper ;;
-        vb) write_vb_wrapper ;;
-    esac
-
-    # Remove the user code file (copy)
-    rm "${INTERNAL_RUNTIME_ENTRYPOINT}"
-
```

Comment:
How is this working with all of this removed 🤔

Reply:
Developer must do this in their code now. No more magic. All documented in examples and notes in README.md